proto/viam/app/v1/robot.proto

```diff
@@ -88,6 +88,8 @@ message ProcessConfig {
   string cwd = 4;
   bool one_shot = 5;
   bool log = 6;
+  int32 stop_signal = 7;
```

Comment:
opt: makeit an enum

Reply:
I'd rather not enum it at the proto level. On a few weird architectures (e.g. if we ever supported some MIPS or SPARC board for some reason), some of the signals are different, so an int would still let something pass the "true" signal without needing a proto change, and the enum could filter at the application level.

Reply:
good point.